pkg/batcher/describeinstances.go

```diff
@@ -73,29 +77,34 @@ func execDescribeInstancesBatch(ec2api ec2iface.EC2API) BatchExecutor[ec2.Descri
 
 		// Execute fully aggregated request
 		// We don't care about the error here since we'll break up the batch upon any sort of failure
-		_ = ec2api.DescribeInstancesPagesWithContext(ctx, firstInput, func(dio *ec2.DescribeInstancesOutput, b bool) bool {
-			for _, r := range dio.Reservations {
-				for _, instance := range r.Instances {
-					missingInstanceIDs.Delete(*instance.InstanceId)
-
-					// Find all indexes where we are requesting this instance and populate with the result
-					for reqID := range inputs {
-						if *inputs[reqID].InstanceIds[0] == *instance.InstanceId {
-							inst := instance // locally scoped to avoid pointer pollution in a range loop
-							results[reqID] = Result[ec2.DescribeInstancesOutput]{Output: &ec2.DescribeInstancesOutput{
-								Reservations: []*ec2.Reservation{{
-									OwnerId:       r.OwnerId,
-									RequesterId:   r.RequesterId,
-									ReservationId: r.ReservationId,
-									Instances:     []*ec2.Instance{inst},
-								}},
-							}}
-						}
+		output, err := ec2api.DescribeInstances(ctx, firstInput)
```

Comment:
AWS SDK Go v2 ships with a pagination helper -- can we use this to still paginate? https://aws.github.io/aws-sdk-go-v2/docs/making-requests/#using-paginators

We should validate this generally -- if we expect results to be returned in a paginated way -- we need to make sure that we iterate through all of the pages